OneCommunityGlobal/HighestGoodNetworkApp
# Imran edit summary submit requirement others

# Description
Gives the user permission to change the requirement to the user to submit a summary.

## Related PRS (if any):
This frontend PR is related to the [PR#942](https://github.com/OneCommunityGlobal/HGNRest/pull/942) backend PR.

## Main changes explained:
- added updateSummaryRequirements to permissionList
- added can edit check on WeeklySummaryOptions in VolunteeringTimeTab.jsx

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Follow steps in the video below

## Screenshots or videos of changes:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/87269725/616a7892-7dc2-4e5d-b3d7-ae1df1d7b514



## Note:
Include the information the reviewers need to know.
